gflownet/gflownet.py

```diff
@@ -81,8 +81,8 @@ def __init__(
             print("Unkown loss. Using flowmatch as default")
             self.loss = "flowmatch"
             self.Z = None
-        if not sample_only:
-            self.loss_eps = torch.tensor(float(1e-5)).to(self.device)
+        # if not sample_only:
+        self.loss_eps = torch.tensor(float(1e-5)).to(self.device)
```

Comment:
Initially this was declared only if `sample_only` is False. I think `loss_eps` is used irrespective of that?